src/main/java/Duke.java

```diff
@@ -1,10 +1,216 @@
+        try {
+            switch (taskType[0]) {
+            case TODO:
+                taskTypeLength = 4;
+                if (taskType.length == 1) {
+                    throw new DukeException();
+                }
+                return new Todo(input.substring(taskTypeLength + 1));
+            case DEADLINE:
+                taskTypeLength = 8;
+                if(taskType.length == 1 || !input.contains("/by")) {
+                    throw new DukeException();
+                }
+                int indexOfBy = input.indexOf("/");
+                return new Deadline(input.substring(taskTypeLength + 1, indexOfBy - 1),
+                        input.substring(indexOfBy + 4));
+            case EVENT:
+                taskTypeLength = 5;
+                if(taskType.length == 1 || !input.contains("/at")) {
+                    throw new DukeException();
+                }
+                int indexOfAt = input.indexOf("/");
+                return new Event(input.substring(taskTypeLength + 1, indexOfAt - 1),
+                        input.substring(indexOfAt + 4));
+            default:
+                printInvalidMeaning();
+                return null;
+            }
+        } catch (DukeException e) {
+            printCannotBeEmpty(taskType);
+            return null;
+        }
+    }
+
+    public static void deleteTask(ArrayList<Task> tasks, int index) {
+        printHorLine();
+        System.out.println("     Noted. I've removed this task: \n" + "       " +
+                tasks.get(index).toString());
+        tasks.remove(tasks.get(index));
+        System.out.println("\n     Now you have " +
+                tasks.size() + " tasks in the list.");
+        printHorLine();
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        retrieveFromFile(FILE_PATH);
+        Scanner in = new Scanner(System.in);
+
+        printHelloMessage();
+
+        String userInput = in.nextLine();
+        while (!userInput.equals(BYE)) {
+            if (userInput.equals(LIST)) {
+                printList(tasks);
+            } else if (userInput.startsWith(DONE)) {
+                try {
+                    int indexOfNumber = 5;
+                    int indexOfTask = Integer.parseInt(userInput.substring(indexOfNumber)) - 1;
+                    markAsDone(tasks.get(indexOfTask));
+                    writeToFile(FILE_PATH);
+                } catch (NumberFormatException e) {
+                    printHorLine();
+                    System.out.println("     ☹ OOPS!!! Please input the number of the task you've completed.");
+                    printHorLine();
+                }
+            } else if (userInput.startsWith(DELETE)) {
```

Comment:
You should improve the level of abstraction here. Consider to create methods for each elseif...